docs/guide/db-query-builder.md

```diff
@@ -620,6 +620,51 @@ $query = (new \yii\db\Query())
 The anonymous function takes a parameter `$row` which contains the current row data and should return a scalar
 value which will be used as the index value for the current row.
 
+## Dealing with table aliases <span id="aliases"></span>
+
+Some use cases of query builder are to create an initial [[yii\db\Query]] object in one place and adjust
+it in other places of the application by adding further conditions or to change options such as sorting.
+When more than one table is used in the query, you may need to disambiguate column names to be explicit about
+which column a condition refers to, e.g. `user.id` vs `post.id`.
+
+Since version 2.0.7 Yii provides methods that help you with disambiguating column names without the need
```

Comment:
no, it's plural. "multiple methods that help you" vs. "one method that helps you"